packages/test/test-end-to-end-tests/src/test/rootDatastores.spec.ts

```diff
@@ -439,13 +422,13 @@ describeFullCompat("Named root data stores", (getTestObjectProvider) => {
 
 			// Should be able to retrieve root datastore from remote
 			assert.doesNotThrow(
-				async () => getAliasedDataStoreEntryPoint(dataObject2, _alias),
+				async () => getAliasedDataStoreEntryPoint(dataObject2, alias),
```

Comment:
Do you also want to check for undefined?

Maybe update that helper function to do the try-catch and convert errors to undefined so you can just treat it like the new API shape?

Reply:
Updated the `getAliasedDataStoreEntryPoint` function to throw error on undefined and do assert.doesNotReject here.